### PR TITLE
makes the networked fibers blob color not look as a dead blob

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
:cl:
tweak: networked fibers color is now dark grey
/:cl:
the old color (almost white) was too similar to a dead blob

new color: 
www

### CODE REVIEW

**Comment by Fikou** [comment]
thats brown

**Comment by ExcessiveUseOfCobblestone** [comment]
we get it tlat you're colorblind

**Comment by wesoda25** [comment]
thats dark grey

**Comment by ExcessiveUseOfCobblestone** [comment]
https://www.htmlcsscolor.com/hex/4F4441

btfo

**Comment by nemvar** [comment]
That's brown.

**Comment by optimumtact** [comment]
What does it look like in game?

**Comment by Tlaltecuhtli** [comment]
weed(2)

**Comment by Shadowflame909** [comment]
If this is the EMP blob. I'm a fan of aqua.

Dabadeedabadie.

File: code/modules/antagonists/blob/blobstrains/networked_fibers.dm [hunk 1]
```diff
@@ -29,7 +29,7 @@
 /datum/reagent/blob/networked_fibers
 	name = "Networked Fibers"
 	taste_description = "efficiency"
-	color = "#CDC0B0"
+	color = "#4F441"
```
Comment (SpaceManiac) [inline code/modules/antagonists/blob/blobstrains/networked_fibers.dm]:
```suggestion
	color = "#4F4441"
```

**Review by SpaceManiac — CHANGES_REQUESTED** [review]

**Review by SpaceManiac — APPROVED** [review]

**Pull request merged**